packages/libs/react-ui/src/styles/tokens/styles.css.ts

```diff
@@ -0,0 +1,309 @@
+/* This file is generated automatically by Design Sync. Do not edit this file directly. */
+
+import { style } from '@vanilla-extract/css';
+
+import { tokens } from './contract.css';
+
+export const fontH1Regular = style({
+  fontFamily: tokens.kda.foundation.typography.family.headingFont,
+  fontSize: tokens.kda.foundation.size.n10,
+  fontWeight: tokens.kda.foundation.typography.weight.headingFont.light,
+  lineHeight: tokens.kda.foundation.size.n14,
+  letterSpacing: tokens.kda.foundation.size.n1,
+  '@media': {
+    '(width >= 1024px)': {
```

Comment:
Can we use rems instead here? 
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

Reply:
The breakpoints are part of the json as well so we can change them ofc, but also could be done independently of the this PR since those are generated files